magprime/templates/magprime_dept_checklist/allotments.html

```diff
@@ -0,0 +1,191 @@
+{% extends "base-admin.html" %}
```

Comment:
I _think_ all this stuff was moved, after this PR was created, into https://github.com/magfest/magfest/